woocommerce/woocommerce-android
# Issue/1750 aztec design changes

Fixes #1750 by adding design changes to the Aztec Toolbar editor. 

~**Note that this PR is in draft till this [Aztec PR](https://github.com/wordpress-mobile/AztecEditor-Android/pull/900) is merged and then we would need to update the hash to this branch before it can be merged.**~

I decided to implement this change in the dark mode branch since it updates styles and colors to the Aztec editor. 

### Screenshots
#### Light Mode:
<img src="https://user-images.githubusercontent.com/22608780/78570237-5526c400-7842-11ea-81fc-c121b6eb8313.png" width="200"/>. <img src="https://user-images.githubusercontent.com/22608780/78570124-2f99ba80-7842-11ea-8511-7ee1e27f2afd.png" width="200"/>. <img src="https://user-images.githubusercontent.com/22608780/78570122-2dcff700-7842-11ea-859f-b3ecf6deed07.png" width="200"/>

#### Dark mode
<img src="https://user-images.githubusercontent.com/22608780/78570367-843d3580-7842-11ea-9861-b1e80fd5af5c.png" width="200"/>. <img src="https://user-images.githubusercontent.com/22608780/78570365-830c0880-7842-11ea-8c46-52e6b0476a81.png" width="200"/>. <img src="https://user-images.githubusercontent.com/22608780/78570357-81424500-7842-11ea-8061-fc0c3f7d93f2.png" width="200"/>

### To test
- Testing in API 28, 28, 21 and 23 with both dark and light mode would be great.
- General smoke testing the Aztec toolbar to ensure that everything is working as expected.

Update release notes:
